src/b-splines/constant.jl

```diff
@@ -31,6 +55,11 @@ end
 """
 Constant b-splines are *nearest-neighbor* interpolations, and effectively
 return `A[round(Int,x)]` when interpolating.
+
+`Constant{Previous}` interpolates to the previous value thus equivalent
+to `A[floor(Int,x)]`.
+`Constant{Next}` interpolates to the next value thus equivalent
+to `A[ceil(Int,x)]`.
```

Comment:
```suggestion
Constant b-splines are *nearest-neighbor* interpolations, and effectively
return `A[round(Int,x)]` when interpolating without scaling.

`Constant{Previous}` interpolates to the previous value and is thus equivalent
to `A[floor(Int,x)]` without scaling.
`Constant{Next}` interpolates to the next value and is thus equivalent
to `A[ceil(Int,x)]` without scaling.
```

Reply:
I added a remark for the inaccuracies when scaling (as far as I understand them).